examples/cordova-sample/MyApp/www/js/index.js

```diff
@@ -268,6 +268,10 @@ initializePurchasesSDK = function() {
   this.setupShouldPurchasePromoProductListener();
   Purchases.enableAdServicesAttributionTokenCollection();
   this.setupPurchaseButtons();
+  window.addEventListener("onCustomerInfoUpdated", (info) => {
+    console.log("customer info updated!");
+    console.log(info);
```

Comment:
I guess since this is a sample app it's NABD but should we log the entire CustomerInfo?

Reply:
oops, forgot to reply. I think I had this mostly for debugging, but I mean… I don’t see any harm in it?